### PR TITLE
Added "Solidity" to the table of contents.

### PR DESCRIPTION
**Pull request by Luc1d1ty**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Solidity is under the sub-category of Java, I don't know whether it should be or not. I have never heard of the language and from my 10 seconds of searching it seems to be its own programming language, am I correct saying this @Luc1d1ty

**Comment by Luc1d1ty** [comment]
Yes . It's a programming language of it's own , used for writing smart contracts . And it shouldn't be under the sub category of java as they are completely different from each other .

**Comment by davorpa** [comment]
Nice catch

**Review by davorpa — APPROVED** [review]

**Comment by Luc1d1ty** [comment]
You are welcome . As this is my very first 'actual' contribution to a big project on github , I am more than happy . As I added only 2 words , I am really hopping to add more than that near future :)

**Comment by Luc1d1ty** [comment]
If you don't mind I have a question, ( I am not yet that much expert using github )  I am unable to merge pull request . Is this a problem from my side ? if yes then please assist me to solve the problem . Thank you .

**Comment by davorpa** [comment]
> If you don't mind I have a question, ( I am not yet that much expert using github ) I am unable to merge pull request . Is this a problem from my side ? if yes then please assist me to solve the problem . Thank you .

Don't be afraid. There are no problems. 

Normally, pull request are contributions made on other repositories you haven't write access rights

In this repo `Ready to merge` label means aproved by staff and waiting to be merged by one with those permissions. (mainly @eshellman )

**Comment by Luc1d1ty** [comment]
Got it ! Thank you .

**Comment by eshellman** [comment]
I see what happened - the section was added before Spring, not realizing that was in the java section.

Not that we need to also move the list out of the java section, not only the heading.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2279,6 +2274,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Snap! Reference Manual](https://snap.berkeley.edu/snapsource/help/SnapManual.pdf) - B. Harvey, J. Mönig (PDF)
 
 
+#### Solidity
```
Comment (CleanMachine1) [inline books/free-programming-books-langs.md:2277]:
```suggestion
### Solidity
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**